templates/vue/src/components/TapestryApp.vue

```diff
@@ -229,29 +233,33 @@ export default {
 
 <style lang="scss" scoped>
 #app-container {
+  position: relative;
   transform: scale(1);
   transform-origin: top left;
   transition: all 0.2s ease-out;
+  width: 100%;
   z-index: 0;
 
   @media screen and (min-width: 500px) {
     &.sidebar-open {
-      transform: scale(0.7);
+      width: calc(100% - max(340px, 30%));
```

Comment:
Just a note on using `width` instead of `transform` here. Animating an element's width triggers an [expensive browser process called reflow](https://pqina.nl/blog/animating-width-and-height-without-the-squish-effect/), which can cause jittery animations for slower computers and/or larger Tapestries.

That being said, the sidebar opening/closing isn't a frequent animation so it might not be that big of a deal. I noticed you wrote the commit @wynnset, so if it's all good for you then it's ok with me.

Reply:
I see. I couldn't find another way to make the drag-select selection rectangle appear in the correct size. If we can find a way to keep this as `transform` and make the drag-select selection work, that would be great. Do you think you could give it a look yourself quickly and let me know if you can find a way @narendrasss ?

Reply:
I tried out a few things for around 30 mins and couldn't find an easy workaround. Might be a case of pre-optimization here; I wouldn't spend too long on it because it's only a brief animation.